server/src/main/java/org/candlepin/dto/rules/v1/OwnerDTO.java

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright (c) 2009 - 2018 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package org.candlepin.dto.rules.v1;
+
+import org.candlepin.dto.CandlepinDTO;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlRootElement;
+
+/**
+ * A DTO representation of the Owner entity as used by the Rules framework.
+ */
+@XmlRootElement
+@XmlAccessorType(XmlAccessType.PROPERTY)
+public class OwnerDTO extends CandlepinDTO<OwnerDTO> {
```

Comment:
You sure we won't need id, name, and/or account here?

Reply:
'defaultServiceLevel' is the only owner field I see being accessed in rules.js, so I'm pretty sure, unless I'm missing something else.